_episodes/14-supplemental-vscode.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Supplemental: Using Git from VS-Code"
+teaching: 10
+exercises: 0
+questions:
+- "How can I use Git with VS-Code?"
+objectives:
+- "Understand how to use Git from VS-Code."
+keypoints:
+- "Using VS-Code's Git integration allows you to version control a project over time."
```

Comment:
This section is a good opportunity to introduce some software development ideas that will be new to interns outside of computer science without needing to make it much longer.
- IDEs are a useful tool because...
	- VS-Code is a freely available and commonly used IDE.
	- Many software developers and researchers prefer to use Git through an IDE.